var/spack/repos/builtin/packages/git-fat-git/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+# Author: Matteo Giantomassi <matteo.giantomassiNOSPAM AT uclouvain.be>
+# Date: October 11, 2016
+from spack import *
+
+
+class GitFatGit(Package):
+    """Simple way to handle fat files without committing them to git, supports
+       synchronization using rsync"""
+
+    homepage = "https://github.com/jedbrown/git-fat"
+    git      = "https://github.com/jedbrown/git-fat.git"
+
+    version('e1733b1c7c4169d0a1d31cb76f168fb0880176c0', commit='e1733b1c7c4169d0a1d31cb76f168fb0880176c0')
```

Comment:
My recommendation for packages without a versioning system is to use the date of the commit as the version number, in the format `2018-08-21`. This allows versions to be sorted, whereas commit hashes aren't sortable. See https://spack.readthedocs.io/en/latest/packaging_guide.html#date-versions.